komga-webui/src/locales/en.json

```diff
@@ -143,19 +143,19 @@
       "vertical": "Vertical",
       "webtoon": "Webtoon"
     },
-    "tooltip_incognito": "Read progress will not be saved"
+    "tooltip_incognito": "Reading progress will not be saved"
   },
   "browse_book": {
-    "comment": "COMMENT",
```

Comment:
All these are errors(?)

Reply:
Those are displayed at the bottom of the book details screen, the case is on purpose.